Add Dayforce docs (#1470)
wso2/docs-ei: en/micro-integrator/docs/references/connectors/ceridiandayforce-connector/ceridiandayforce-connector-config.md

# Setting up the Ceridian Dayforce Environment

The Dayforce Connector allows you to access the REST API of 
[Ceridian Dayforce HCM](https://www.ceridian.com/products/dayforce), 
which lets you store your human capital information and retrieve them back when needed. 

> To use the Dayforce cloud service, you must have an [Dayforce HCM](https://www.dayforcehcm.com) account. 

Reviewer: ```suggestion
To use the Dayforce cloud service, you must have a [Dayforce HCM](https://www.dayforcehcm.com) account. 
```

> To test the REST API of dayforce we will use a Dayforce developer acount which is free and lets us access a 

Reviewer: ```suggestion
To test the REST API of Dayforce we will use a Dayforce developer account, which is free and lets us access a 
```

developer Dayforce instance.

## Signing Up for Dayforce Developer Account

* **To sign up for Dayforce Developer Account:**

    1. Navigate to [Ceridian Dayforce Developer Network](https://developers.dayforce.com) and select **Register**.
    2. Follow the online instructions.

> If your companyhas already purchased a namespace in Dayforce use that to sign up. Otherwise you can still use their 

Reviewer: ```suggestion
If your company has already purchased a namespace in Dayforce, use that to sign up. Otherwise, you can still use their 
```
